isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.cpp

```diff
@@ -782,6 +785,144 @@ namespace Isis {
     return QString(m_ui->inputControlNetCombo->currentText());
   }
 
```

Comment:
Make sure to document the updateBundleObservationSolveSettings method here

Reply:
Added a history entry + function documentation for the method.